frontend/src/routes/(frontend)/explore/layout.tsx

```diff
@@ -54,3 +52,35 @@ export default component$(() => {
 		</div>
 	)
 })
+
+export const activeClasses = [
+	'relative',
+	'before:block',
+	'before:content-[""]',
+	'before:absolute',
+	'before:w-0',
+	'before:h-0',
+	'before:bottom-[-1px]',
+	'before:left-1/2',
+	'before:translate-x-[-50%]',
+	'before:border-solid',
+	'before:border-t-0',
+	'before:border-x-[0.7rem]',
+	'before:border-b-[0.7rem]',
+	'before:border-x-transparent',
+	'before:border-b-wildebeest-500',
+	'after:block',
```

Comment:
I would have liked to define the repeating classes only once and combined generated them dynamically, unfortunately is seems like you need to have the full class present for tailwind to pick it up 😓  (/you can construct the class names dynamically)

https://tailwindcss.com/docs/content-configuration#dynamic-class-names